kubernetes/README.md

```diff
@@ -245,30 +248,41 @@ Note that these are the recommended `requests` and `limits` values for both the
 <details>
   <summary>High Volume Deployment</summary>
 
-  ```yaml
-  requests:
-    cpu: 500m
-    memory: 512M
+```yaml
+requests:
+  cpu: 500m
+  memory: 512M
 
-  limits:
-    cpu: 1000m
-    memory: 1024M
-  ```
+limits:
+  memory: 1024M
+```
 
 </details>
 
 <details>
   <summary>Very High Volume Deployment</summary>
 
-  ```yaml
-  requests:
-    cpu: 1000m
-    memory: 1024M
+The best practices for Kubernetes resource limits are:
+
+1.  Never set [CPU](https://kubernetes.io/docs/tasks/configure-pod-container/assign-cpu-resource) limits to avoid Kubernetes CPU throttling and allow for efficient use of available CPU.
+2.  Set [Memory](https://kubernetes.io/docs/tasks/configure-pod-container/assign-memory-resource) limits same as requests to help avoiding OOM kills.
+
+This works well in the deployment scenario where the 1Password SCIM bridge is the only pod in the cluster, so that it has a priority of service and can consume all the resources in the cluster without affecting the performance of other pods.
 
-  limits:
-    cpu: 2000m
-    memory: 2048M
-  ```
+If you are deploying the SCIM bridge to a cluster with other production services, we recommend using the following resources depending on the size of your deployment.
+
+> **Note**
+>
+> We are recommending not to set CPU limits when you expect performance. However you can set CPU limits if you have a good usecase to manage resources. It can be modified in your deployment based on our generic example and its expected to work in most circumstances.
```

Comment:
```suggestion
> We recommend to not set CPU limits when you expect higher performance. However, you can set CPU limits to manage resources if needed. This can be modified in your deployment based on our generic example, and its expected to work in most circumstances.
```